doc/extending/creating_a_numba_jax_op.rst

```diff
@@ -626,28 +626,8 @@ Step 4: Write tests
 
 Note
 ----
-In out previous example of extending JAX, :class:`Eye`\ :class:`Op` was used with the test function as follows:
-
-.. code:: python
-
-    def test_jax_Eye():
-        """Test JAX conversion of the `Eye` `Op`."""
-
-        # Create a symbolic input for `Eye`
-        x_at = pt.scalar()
-
-        # Create a variable that is the output of an `Eye` `Op`
-        eye_var = pt.eye(x_at)
-
-        # Create an PyTensor `FunctionGraph`
-        out_fg = FunctionGraph(outputs=[eye_var])
-
-        # Pass the graph and any inputs to the testing function
-        compare_jax_and_py(out_fg, [3])
-
-This one nowadays leads to a test failure due to new restrictions in JAX + JIT,
-as reported in issue `#654 <https://github.com/pymc-devs/pytensor/issues/654>`_.
-All jitted functions now must have constant shape, which means a graph like the
+Due to new restrictions in JAX JIT as reported in issue `#654 <https://github.com/pymc-devs/pytensor/issues/654>`_,
+all jitted functions now must have constant shape. In other words, only PyTensor graphs with static shapes
+can be translated to JAX at the moment. It means a graph like the
```

Comment:
I think an example here is useful. It's not that any graph with eye can't be translated. If it has constant inputs is fine, or if it depends on the shape of another variable.